### ISSUE TITLE
python setup.py bdist_wheel error

### ISSUE THREAD

**Issue by QYWT** [traveller59/spconv#272]
I encountered an error during compilation.
cunda 10.1.243
pytorch 1.5.0
torchvision  0.6.0 
cmake 3.16.0

The error was initially reported as
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
CUDA_CUBLAS
    linked by target "cuhash" in directory /home/ly/PCdet/OpenPCDet/spconv/src/cuhash
    linked by target "spconv" in directory /home/ly/PCdet/OpenPCDet/spconv/src/spconv

After adjusting the pytorch version, the error was  reported as

python setup.py bdist_wheel
running bdist_wheel
running build
running build_py
running build_ext
Release
|||||CMAKE ARGS||||| ['-DCMAKE_PREFIX_PATH=/home/ly/anaconda3/envs/PCDet/lib/python3.6/site-packages/torch', '-DPYBIND11_PYTHON_VERSION=3.6', '-DSPCONV_BuildTests=OFF', '-DPYTORCH_VERSION=10500', '-DCMAKE_CUDA_FLAGS="--expt-relaxed-constexpr" -D__CUDA_NO_HALF_OPERATORS__ -D__CUDA_NO_HALF_CONVERSIONS__', '-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=/home/ly/OpenPCDet-master/spconv/build/lib.linux-x86_64-3.6/spconv', '-DCMAKE_BUILD_TYPE=Release']
-- Caffe2: CUDA detected: 10.1
-- Caffe2: CUDA nvcc is: /usr/local/cuda-10.1/bin/nvcc
-- Caffe2: CUDA toolkit directory: /usr/local/cuda-10.1
-- Caffe2: Header version is: 10.1
-- Found cuDNN: v7.6.5  (include: /usr/local/cuda-10.1/include, library: /usr/local/cuda-10.1/lib64/libcudnn.so)
-- Autodetected CUDA architecture(s):  6.1
-- Added CUDA NVCC flags for: -gencode;arch=compute_61,code=sm_61
-- pybind11 v2.5.0
-- Configuring done
-- Generating done
-- Build files have been written to: /home/ly/OpenPCDet-master/spconv/build/temp.linux-x86_64-3.6
[ 20%] Built target spconv_utils
[ 30%] Linking CXX shared library ../../../lib.linux-x86_64-3.6/spconv/libspconv.so
/usr/bin/ld: not found -lCUDA_cublas_LIBRARY-NOTFOUND
collect2: error: ld returned 1 exit status
src/spconv/CMakeFiles/spconv.dir/build.make:190: recipe for target '../lib.linux-x86_64-3.6/spconv/libspconv.so' failed
make[2]: *** [../lib.linux-x86_64-3.6/spconv/libspconv.so] Error 1
CMakeFiles/Makefile2:135: recipe for target 'src/spconv/CMakeFiles/spconv.dir/all' failed
make[1]: *** [src/spconv/CMakeFiles/spconv.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
Traceback (most recent call last):
  File "setup.py", line 108, in <module>
    zip_safe=False,
  File "/home/ly/anaconda3/envs/PCDet/lib/python3.6/site-packages/setuptools/__init__.py", line 153, in setup
    return distutils.core.setup(**attrs)
  File "/home/ly/anaconda3/envs/PCDet/lib/python3.6/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/home/ly/anaconda3/envs/PCDet/lib/python3.6/distutils/dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "/home/ly/anaconda3/envs/PCDet/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/home/ly/anaconda3/envs/PCDet/lib/python3.6/site-packages/wheel/bdist_wheel.py", line 299, in run
    self.run_command('build')
  File "/home/ly/anaconda3/envs/PCDet/lib/python3.6/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/home/ly/anaconda3/envs/PCDet/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/home/ly/anaconda3/envs/PCDet/lib/python3.6/distutils/command/build.py", line 135, in run
    self.run_command(cmd_name)
  File "/home/ly/anaconda3/envs/PCDet/lib/python3.6/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/home/ly/anaconda3/envs/PCDet/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "setup.py", line 48, in run
    self.build_extension(ext)
  File "setup.py", line 92, in build_extension
    subprocess.check_call(['cmake', '--build', '.'] + build_args, cwd=self.build_temp)
  File "/home/ly/anaconda3/envs/PCDet/lib/python3.6/subprocess.py", line 311, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['cmake', '--build', '.', '--config', 'Release', '--', '-j4']' returned non-zero exit status 2.

**Comment by QYWT** [comment]
I solved this problem by reinstalling cuda and cudnn

**Comment by rancheng** [comment]
In case anyone might have this error:
```sh
In file included from /usr/include/c++/7/ext/string_conversions.h:41:0,
                 from /usr/include/c++/7/bits/basic_string.h:6361,
                 from /usr/include/c++/7/string:52,
                 from /usr/include/c++/7/bits/locale_classes.h:40,
                 from /usr/include/c++/7/bits/ios_base.h:41,
                 from /usr/include/c++/7/ios:42,
                 from /usr/include/c++/7/ostream:38,
                 from /usr/include/c++/7/iostream:39,
                 from /home/ran/Documents/project/spconv/include/tensorview/common.h:15,
                 from /home/ran/Documents/project/spconv/include/tensorview/tensorview.h:16,
                 from /home/ran/Documents/project/spconv/include/cuhash/definitions.h:23,
                 from /home/ran/Documents/project/spconv/include/cuhash/debugging.h:21,
                 from /home/ran/Documents/project/spconv/src/cuhash/hash_table.cpp:17:
/usr/include/c++/7/cstdlib:75:15: fatal error: stdlib.h: No such file or directory
 #include_next <stdlib.h>
               ^~~~~~~~~~
compilation terminated.
src/cuhash/CMakeFiles/cuhash.dir/build.make:94: recipe for target 'src/cuhash/CMakeFiles/cuhash.dir/hash_table.cpp.o' failed
make[2]: *** [src/cuhash/CMakeFiles/cuhash.dir/hash_table.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from /usr/local/cuda-10.0/targets/x86_64-linux/include/crt/math_functions.h:8863:0,
                 from /usr/local/cuda-10.0/targets/x86_64-linux/include/crt/common_functions.h:267,
                 from /usr/local/cuda-10.0/targets/x86_64-linux/include/cuda_runtime.h:120,
                 from <command-line>:0:
/usr/include/c++/7/cmath:45:15: fatal error: math.h: No such file or directory
 #include_next <math.h>
               ^~~~~~~~
compilation terminated.
src/utils/CMakeFiles/spconv_nms.dir/build.make:81: recipe for target 'src/utils/CMakeFiles/spconv_nms.dir/nms.cu.o' failed
make[2]: *** [src/utils/CMakeFiles/spconv_nms.dir/nms.cu.o] Error 1
CMakeFiles/Makefile2:231: recipe for target 'src/utils/CMakeFiles/spconv_nms.dir/all' failed
make[1]: *** [src/utils/CMakeFiles/spconv_nms.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
In file included from /usr/local/cuda-10.0/include/crt/math_functions.h:8863:0,
                 from /usr/local/cuda-10.0/include/crt/common_functions.h:267,
                 from /usr/local/cuda-10.0/include/cuda_runtime.h:120,
                 from <command-line>:0:
/usr/include/c++/7/cmath:45:15: fatal error: math.h: No such file or directory
 #include_next <math.h>
               ^~~~~~~~
compilation terminated.
src/cuhash/CMakeFiles/cuhash.dir/build.make:81: recipe for target 'src/cuhash/CMakeFiles/cuhash.dir/hash_functions.cu.o' failed
make[2]: *** [src/cuhash/CMakeFiles/cuhash.dir/hash_functions.cu.o] Error 1
In file included from /usr/local/cuda-10.0/include/crt/math_functions.h:8863:0,
                 from /usr/local/cuda-10.0/include/crt/common_functions.h:267,
                 from /usr/local/cuda-10.0/include/cuda_runtime.h:120,
                 from <command-line>:0:
/usr/include/c++/7/cmath:45:15: fatal error: math.h: No such file or directory
 #include_next <math.h>

```

I'm compiling the environment on `gcc-7`, I just modify those files:

```
/usr/include/c++/7/cmath
/usr/include/c++/7/bits/std_abs.h
/usr/include/c++/7/cstdlib
```

and replace all those `#include_next <*>` with `#include<*>` to compile, after compile finished, don't forget to revert all those files back.
